tensorflow/core/ops/image_ops.cc

```diff
@@ -925,4 +925,38 @@ REGISTER_OP("CombinedNonMaxSuppression")
     .Attr("clip_boxes: bool = true")
     .SetShapeFn(CombinedNMSShapeFn);
 
+REGISTER_OP("GenerateBoundingBoxProposals")
+    .Input("scores: float")
+    .Input("bbox_deltas: float")
+    .Input("image_info: float")
+    .Input("anchors: float")
+    .Output("rois: float")
+    .Output("roi_probabilities: float")
+    .Attr("pre_nms_topn: int = 6000")
```

Comment:
Why are these attributes and not inputs?

Reply:
Because they don't depend on data? They are model parameters and fixed at the model construction time.

Reply:
I converted these to tensors as requested. But considering the op, it will just cause an extra overhead since it will require many constant ops to be scheduled unnecessarily.